examples/dummy-iiostream.c

```diff
@@ -8,7 +8,7 @@
  *
  * How to setup the sample IIO dummy device and hrtimer trigger:
  *
- *   1. sudo modprobe industrialio kfifo_buf
+ *   1. sudo modprobe industrialio kfifo_buf industrialio-sw-trigger
  *   2. sudo modprobe iio_dummy iio-trig-hrtimer
```

Comment:
Since this patch is a cleanup patch for two of the earlier patches in this series do you mind splitting this up and squashing the cleanup into its respective patch (git rebase -i). Thanks.